ExecCpp/Make.PeleC

```diff
@@ -151,6 +170,21 @@ AMReX_buildInfo.cpp:
 
 include $(AMREX_HOME)/Tools/GNUMake/Make.rules
 
+ifeq ($(USE_SUNDIALS_PP),TRUE)
```

Comment:
Is this necessary? Don't we do sundials as a submodule now?

Reply:
This is an ongoing irritation.  For the foreseeable future, if the Chemistry_Model == Null, sundials should not be needed by any of the Pele codes, otherwise it should be needed by all of them.  I'm thinking that USE_SUNDIALS_PP should be replaced everywhere by PELE_USE_CHEM, and that should be set FALSE if Chemistry_Model==Null, and TRUE otherwise.

Can anyone think of objections to this strategy?

Reply:
I have no objections. I would say that we should see how it is currently done in PeleC (I think this differs from that but I don't tend to deal with that stuff so I may be wrong). And then do what you suggest but in a separate PR...

Reply:
Since I use both PeleC and PeleLM, I don't keep things in submodules. I hadn't realized this stuff differed from the development branch because I added it so long ago. I will remove it for now and we can figure it out separately.

Reply:
This change has to come also from pelephysics side, its build system has to be updated. There are couple others like CVODE_BOXINTEG that is unnecessary. guess this is a separate discussion from this PR.

Reply:
Yes, we'll clean that up in a separate PR for PelePhysics that will then trigger edits in PeleLM and PeleC